speclang_modern.tex

```diff
@@ -2349,16 +2354,16 @@ \subsection{Memory blocks and pointer dereferencing}
 following built-in predicates deal with this notion:
 \begin{itemize}
 \item \valid\indexttbs{valid} applies to a set of terms 
-(see Section~\ref{sec:locations}) of some pointer type. 
+(see Section~\ref{sec:locations}) of some pointer type (other than \lstinline|void*|). 
```

Comment:
I think the formulation "non-void pointer type", or "(non-void) pointer type" (with parentheses) would be more concise. It's not used very often, but there used to be [a Frama-C error message](https://lists.gforge.inria.fr/pipermail/frama-c-discuss/2013-December/004165.html) that included it.

Reply:
@maroneze "non-void pointer type" does not seem completely standard. "Pointer to non-void type" would be better in this respect, but I don't feel that it makes the sentence clearer.